### PR TITLE
[hikogui] update to version 0.7.0

### PR DESCRIPTION
**Pull request by takev**
Update hikogui to version 0.7.0.
This library was called ttauri, which is a port in vcpkg already, but was renamed.

The port files where simply copied and modified for the new version, therefor the port file may have old statements in them.

- #### What does your PR fix?
no fixes.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
no changes: x64-windows-static

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by JonLiu1993** [comment]
@takev, thanks for the PR, could you update ttauri instead of adding a new port of hikogui?

**Comment by dg0yt** [comment]
> @takev, thanks for the PR, could you update ttauri instead of adding a new port of hikogui?

We need both ports: `hikogui` as proposed, and `ttauri` as an empty port depending on `hikogui`. Cf. `proj` vs. `prj4`.

**Comment by takev** [comment]
@dg0yt I have updated the pull request to make ttauri empty and depended on hikogui.
It does not make much sense to create polyfills, as many things have changed. Also both ttauri and hikogui are still pre-version-1.

File: ports/hikogui/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hikogui",
+  "version": "0.7.0",
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/hikogui/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/hikogui/vcpkg.json:4]: We should remove `port-version` when we add a new port

File: versions/h-/hikogui.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "git-tree": "6d88971773847339a04ec50311d8b40eaf0f7982",
+      "version": "0.7.0",
+      "port-version": 2
+    },
+    {
+      "git-tree": "708ad193cf5264aeb0738c9e12069bae5c816597",
+      "version": "0.7.0",
+      "port-version": 1
+    }
+  ]
```
Comment (JonLiu1993) [inline versions/h-/hikogui.json:13]:
```suggestion
```

Comment (JonLiu1993) [reply versions/h-/hikogui.json:13]: We just need to add one git-tree record.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/h-/hikogui.json (hunk 2 above)
Comment (JonLiu1993) [inline versions/h-/hikogui.json:13]:
```suggestion
```

Comment (JonLiu1993) [reply versions/h-/hikogui.json:13]: Please remove this code and run command "./vcpkg x-add-version --all --overwrite-version" and commit again.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/h-/hikogui.json (hunk 2 above)
Comment (takev) [reply versions/h-/hikogui.json]: Done.

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**